content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -256,6 +349,73 @@ System.out.println(map.lowerKey(3));     // ERROR
 
 </JavaSection>
 
+<PySection>
+
+Python does not have built in sorted maps or sorted sets in it's standard
+library. However, sorted maps in Python can be created by adding a dictionary to
+a sorted set, where each element in the sorted set is a key in the dictionary
+and values can be assigned with the dictionary.
+
+This is the most straightforward implementation, and the ground up
+implementation of a sorted set can be found in the above section.
+
+Since some online judges include additional libraries, an implementation of
+`SortedDict` from the `sortedcontainers` library can be found below. All
+operations shown below are $\mathcal{O}(\log N)$ time, except for its
+$\mathcal{O}(N \log N)$ initialization and getting all items, keys, or values
+which is $\mathcal{O}(N)$ time.
+
+```py
+from sortedcontainers import SortedDict
+
+sorted_map = SortedDict({1: "one", 4: "four", 3: "three"})
+print(sorted_map)  # SortedDict({1: 'one', 3: 'three', 4: 'four'})
+
+# Add elements
+sorted_map[2] = "two"
+sorted_map[5] = "five"
+print(sorted_map)  # SortedDict({1: 'one', 2: 'two', 3: 'three', 4: 'four', 5: 'five'})
+
+# Remove elements
+del sorted_map[3]
+print(sorted_map)  # SortedDict({1: 'one', 2: 'two', 4: 'four', 5: 'five'})
+
+# Check if a key is in the sorted dict
+print(1 in sorted_map)  # True
+print(100 in sorted_map)  # False
+
+# Get the key's value
+print(sorted_map[2])  # two
+print(sorted_map[4])  # four
+
+# Get all items (key value pairs), keys, or values
+print(
+	sorted_map.items()
+)  # SortedItemsView(SortedDict({1: 'one', 2: 'two', 4: 'four', 5: 'five'}))
+print(
+	sorted_map.keys()
+)  # SortedKeysView(SortedDict({1: 'one', 2: 'two', 4: 'four', 5: 'five'}))
```

Comment:
perhaps put the comment on top of the line of code so we don't get this wonky formatting

Reply:
Changed to the below format:

```py
# Get all items (key value pairs), keys, or values below

# Output SortedItemsView(SortedDict({1: 'one', 2: 'two', 4: 'four', 5: 'five'}))
print(sorted_map.items())

# Output SortedKeysView(SortedDict({1: 'one', 2: 'two', 4: 'four', 5: 'five'}))
print(sorted_map.keys())

# Output SortedValuesView(SortedDict({1: 'one', 2: 'two', 4: 'four', 5: 'five'}))
print(sorted_map.values())
```